src/comparator/wrappers.py

```diff
@@ -834,51 +818,86 @@ def __init__(
         # set including the nested messages/enums, since they could also be referenced.
         # Key is the full name of the message/enum and value is the Message/Enum object.
         self._get_global_info_map(source_code_locations_map)
-        # Get all messages in the map.
-        # TODO(xiaozhenliu): update the map when global messages map is in place.
-        self.messages_map = self._get_messages_map(source_code_locations_map)
-
+        # Get all **used** information for comparison.
         self.packaging_options_map = defaultdict(dict)
```

Comment:
Instead of putting every message in the messages map, we only extract the first-level message in the API definition files, and the used message fro comparison, this will help us avoid unnecessary breaking change messages.